### ISSUE TITLE
Change success color variables to call proper green variables

### ISSUE THREAD

**Issue by mceledonia** [patternfly/patternfly#2479]
Per #2320 

(excuse my shorthand variable names) 
`success-100` should point to `green-500`
`success-200` should point to `green-700` 
`green-500` should be changed to `#3E8635`

This should be a non-breaking change, it just changes greens to better greens.

@mcoker

**Comment by serenamarie125** [comment]
@rachael-phillips @LHinson this is needed for both Kiali & OpenShift Dev Perspective

**Comment by mcoker** [comment]
We reverted this change in core and are going to hold off until after the holiday break in case issues arise, and until we communicate that we will be updating the global colors or potentially in our next breaking change release.

**Comment by coreyvickery** [comment]
@mceledonia I added this new green to our XD kit.

**Comment by mcoker** [comment]
@mceledonia since we're planning for an upcoming breaking change release, are you OK pushing this out for that release?

**Comment by mceledonia** [comment]
@mcoker Yeah that sounds good to me!

**Comment by mcoker** [comment]
closed via https://github.com/patternfly/patternfly/pull/2817/
